From: Jose Luis Rivero <jrivero@osrfoundation.org>
Date: Wed, 6 May 2026 12:45:00 +0200
Subject: Export only public flags in pkg-config

Forwarded: No
Last-Update: 2026-05-06
---
 cmake/dart.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/cmake/dart.pc.in
+++ b/cmake/dart.pc.in
@@ -9,4 +9,4 @@ Description: @DART_PKG_DESC@
 Version: @DART_VERSION@
 Requires: @DART_PKG_EXTERNAL_DEPS@
 Libs: -L${libdir} -ldart
-Cflags: @CMAKE_CXX_FLAGS@
+Cflags: -I${includedir}
