From 30f6835c74b7d0ddec9828b7d75343619ed4decd Mon Sep 17 00:00:00 2001
From: Jeongseok Lee <jslee02@gmail.com>
Date: Thu, 18 Dec 2025 00:00:36 -0800
Subject: [PATCH] tests: stabilize World::Cloning

---
 tests/integration/test_World.cpp | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/tests/integration/test_World.cpp b/tests/integration/test_World.cpp
index 06ca875bb6459..b4efbddc6e222 100644
--- a/tests/integration/test_World.cpp
+++ b/tests/integration/test_World.cpp
@@ -37,6 +37,7 @@
 #include "dart/dynamics/RevoluteJoint.hpp"
 #include "dart/dynamics/Skeleton.hpp"
 #include "dart/math/Geometry.hpp"
+#include "dart/math/Random.hpp"
 #include "dart/utils/SkelParser.hpp"
 
 #include <gtest/gtest.h>
@@ -185,6 +186,9 @@ TEST(World, AddingAndRemovingSkeletons)
 //==============================================================================
 TEST(World, Cloning)
 {
+  // Seed random generator for deterministic tests
+  Random::setSeed(42);
+
   // Create a list of skel files to test with
   std::vector<common::Uri> fileList;
   fileList.push_back("dart://sample/skel/test/chainwhipa.skel");
@@ -224,10 +228,10 @@ TEST(World, Cloning)
     for (std::size_t j = 1; j < 5; ++j)
       clones.push_back(clones[j - 1]->clone());
 
-#if DART_BUILD_MODE_DEBUG
-    std::size_t numIterations = 3;
-#else
+#if DART_BUILD_MODE_RELEASE
     std::size_t numIterations = 500;
+#else
+    std::size_t numIterations = 3;
 #endif
 
     for (std::size_t j = 0; j < numIterations; ++j) {
